pyproject.toml

```diff
@@ -139,20 +119,37 @@ testing = [
     "pytest-codspeed~=2.2.0; python_version < '3.13'",
     "packaging>=23.2",
     "jsonschema>=4.23.0",
-]
-testing-extra = [
```

Comment:
Hmm, don't we still want to group these?

Reply:
https://github.com/astral-sh/uv/issues/5632